MovieDB/Scenes/Detail/DetailViewModel.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  DetailViewModel.swift
+//  Project2
+//
+//  Created by cuonghx on 6/15/19.
+//  Copyright © 2019 Sun*. All rights reserved.
+//
+
+import RxDataSources
+
+typealias ActorSection = SectionModel<String, Actor>
+typealias CompanySection = SectionModel<String, Company>
+
+struct DetailViewModel {
+    var usecase: DetailUseCaseType
+    var navigator: DetailNavigatorType
+    var movie: Movie
+}
+
+extension DetailViewModel: ViewModelType {
+    struct Input {
+        var triggerLoadActor: Driver<Void>
```

Comment:
```suggestion
        var loadActorTrigger: Driver<Void>
```
tương tự dưới, mà nếu chỉ load cùng 1 thời điểm thì có thể gộp thành 1 cái loadTrigger